config/webpack/webpack.config.js

```diff
@@ -12,6 +12,7 @@ const DuplicatePackageCheckerPlugin = require('@cerner/duplicate-package-checker
 const aggregateTranslations = require('terra-aggregate-translations');
 const ThemeAggregator = require('../../scripts/aggregate-themes/theme-aggregator');
 const getThemeWebpackPromise = require('./getThemeWebpackPromise');
+const ThemePlugin = require('../../scripts/postcss/ThemePlugin');
```

Comment:
If this is a webpack plugin, should it live under `config/webpack/plugins`?

Reply:
Well, technically it's a postcss plugin that the postcss webpack loader uses... 🤷‍♂

Reply:
my vote is to at least move this to the config/webpack to keep these concepts close

Reply:
Ok, it's already kinda weird living in scripts anyway.